test/e2e/common/configmap.go

```diff
@@ -134,6 +135,27 @@ var _ = ginkgo.Describe("[sig-node] ConfigMap", func() {
 		configMap, err := newConfigMapWithEmptyKey(f)
 		framework.ExpectError(err, "created configMap %q with empty key in namespace %q", configMap.Name, f.Namespace.Name)
 	})
+
+	ginkgo.It("should patch ConfigMap successfully", func() {
```

Comment:
this should be using the framework on this call line vs. ginkgo. 

This applies to everything here.

Reply:
we don't have a `framework.It`; we do have a `framework.ConformanceIt` but explicitly disallow new tests from being added with ConformanceIt, they must be promoted into it

Reply:
we don't have a `framework.It`; we do have a `framework.ConformanceIt` but explicitly disallow new tests from being added with ConformanceIt, they must be promoted into it

Reply:
Sorry retract, I'm brain fry, comment below still applies though.